ci/check-client

```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env bash
 set -eux -o pipefail
 
+cd $TRAVIS_BUILD_DIR
 # Exit early if there were no client changes.
-"${TRAVIS_BUILD_DIR}/ci/changes-in-dir" client || exit 0
+.ci/changes-in-dir client || exit 0
```

Comment:
`.ci/changes-in-dir` is not a valid path - you probably meant `./ci/changes-in-dir`. Builds should fail, except `|| exit 0` prevents that from happening and skips everything that comes afterward.